Bottom: 9fd44465a4fa3112dfd2b08fa8f75a8dd2220d93
Top:    e046a0917dbcdabceab41e7ba3f82490a8b7bf17
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-17 15:14:21 +0100

Refresh of bikeshed-commit-11700ba8c3c951

---

diff --git a/src/file.c b/src/file.c
index baf3696..8888432 100644
--- a/src/file.c
+++ b/src/file.c
@@ -1414,7 +1414,7 @@ ReadLibraryContents (void)
   LibraryEntryTypePtr entry;
 
   /* If we don't have a command to execute to find the library contents,
-   * quit now. This is used by default on Windows builds (set in main.c),
+   * skip this. This is used by default on Windows builds (set in main.c),
    * as we can't normally run shell scripts or expect to have m4 present.
    */
   if (*Settings.LibraryContentsCommand != '\0')
